apps/cms/config/sync/language/de/views.view.redirect.yml

```diff
@@ -44,19 +44,19 @@ display:
             label: Statuscode
             group_items:
               1:
-                title: '300 Multiple Choices'
+                title: '300 Mehrere Auswahlmöglichkeiten'
```

Comment:
Drush export shenanigans? I would rather not include this in the diff. Lets keep it english by default.

Reply:
This should be removed now.